CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+      >You will see the list of your current remotes.  
+      >origin which points to your GitHub fork of the project.  
+      >upstream which points to the original project's GitHub repository.            
+
+### To run the project in development mode
+* To install the dependencies, use this command in your terminal:
+
+  ```sh
+       npm install
+  ```
+
+* After the installation process is completed, use this command to run the app in development mode:
+
+  ```sh
+      npm start
+  ```          
+
+## Your first Contribution
+Unsure where to begin contributing to SpeakEd? 😕<br /> 
+Look for issues tagged with [good first issue](https://github.com/HarshCasper/SpeakEd/contribute).
+
+When you contribute to a project it's good practice to create a new branch from the `main` branch. The branch name should be short and meaningful and relevant to the work you are intending to do.
+
+To create a new branch: 
+
+* Ensure that you're on the `main` branch.
+
+  ```sh
+      $ git checkout main
+  ```
+
+* Then use the following command which will sync your local copy with the upstream project and will sync it to your forked repository.
+
+  ```sh
+      $ git pull upstream main && git push origin main
+  ```
+* To create a branch, use this command: 
+
+  ```sh
+      $ git checkout -b <branchname>
+  ```
+Now you can start working to fix the issue. 🚀
+
+## To Create a Pull Request
+
+To create a PR you need to push your branch to the `origin` remote.
+
+1. To push a new branch: 
+
+   ```sh
+       $ git push -u origin <branchname>
+   ```
+   >This will create the branch on your GitHub project and the -u flag links your local branch with remote branch, and you can just use git push origin when commiting any changes in the local branch.
+
+2. Finally, go to your forked repository in your browser and press the green button `Compare & pull request`.  
+   >This will open a page in which you'll be able to further edit the description for your proposed changes.
+
+3. Make sure you follow the Pull Request Template and your PR's description should contain GitHub's special keyword references that automatically close the related issue when the PR is merged. (Check [this](https://github.blog/2013-05-14-closing-issues-via-pull-requests/) for more info)
+
+4. Finally, press the green button labelled as `Create pull request`.
+
+## Learning Resources 📚
+
+* [Git Started with GitHub](https://www.udemy.com/course/git-started-with-github/?LSNPUBID=JVFxdTr9V80&ranEAID=JVFxdTr9V80&ranMID=39197&ranSiteID=JVFxdTr9V80-.eo946O7LA.e_kedPDtXLw&utm_medium=udemyads&utm_source=aff-campaign)
+* [Learn Git: Everything You Need To Know](https://www.udemy.com/course/learngit/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-K5pDC6IWSDnHLwh8jTAbEg&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads)
+* [The beginner's guide to contribute to a GitHub project](https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/)
+
+
+
+
+
+
+
+
```

Comment:
Why are there so many extra spaces?

Reply:
Sorry, I didn't notice it. 
Will fix it.